src/DBEditor/ScriptsSQL/CreateTables/003CreateTables.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE IF NOT EXISTS public.users
+(
+    id uuid NOT NULL DEFAULT gen_random_uuid() UNIQUE,
+    name character varying(50) COLLATE pg_catalog."default" NOT NULL,
+    password character varying(50) COLLATE pg_catalog."default" NOT NULL,
+
+    CONSTRAINT pkey_users  PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS public.items
+(
+    key character varying(50) COLLATE pg_catalog."default" NOT NULL UNIQUE,
+    expiration_period int NOT NULL,
+
```

Comment:
Add additional column "expiration_date"

Reply:
I think we should rename the "expiration_period" to "expiration_date" and not add an additional column.